src/odemis/gui/comp/positions.py

```diff
@@ -0,0 +1,254 @@
+import logging
+import math
+import wx
+from odemis import gui, model
+from odemis.gui.comp.text import UnitFloatCtrl
+
+
+class EditMeteorPositionsDialog(wx.Dialog):
```

Comment:
Adjust this class like https://github.com/delmic/odemis/blob/master/src/odemis/gui/comp/settings.py to have a cleaner init?